.teamcity/performance-test-durations.json

```diff
@@ -1,4 +1,19 @@
 [ {
+  "scenario" : "org.gradle.performance.regression.android.RealLifeAndroidBuildPerformanceTest.clean assembleDebug with clean transforms cache",
+  "durations" : [ {
+    "testProject" : "largeAndroidBuild",
+    "linux" : 2406000
+  }, {
+    "testProject" : "santaTrackerAndroidBuild",
+    "linux" : 3659000
+  } ]
+}, {
+    "scenario" : "org.gradle.performance.regression.android.RealLifeAndroidBuildPerformanceTest.clean phthalic:assembleDebug with clean transforms cache",
+    "durations" : [ {
+        "testProject" : "largeAndroidBuild",
+        "linux" : 1775000
+    } ]
+}, {
```

Comment:
Why did the order change here?

Reply:
Because `RealLifeAndroidBuildPerformanceTest` is before `RealLifeAndroidBuildSlowPerformanceTest`. And I moved the test between those two test classes.

Reply:
👍 missed that, thanks.